pkg/autodiscovery/listeners/docker.go

```diff
@@ -137,11 +141,18 @@ func (l *DockerListener) init() {
 		}
 		var svc Service
 
+		checkNames, err := getCheckNamesFromLabels(co.Labels)
+		if err != nil {
+			log.Errorf("Error getting check names from docker labels: %v", err)
+			checkNames = nil
+		}
+
```

Comment:
I think the `getCheckNamesFromLabels` return already `checkNames = nil` in case of error

Reply:
yes